tests/AboutPromiseTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace seregazhuk\React\PromiseTesting\tests;
+
+use Exception;
+use React\Promise\Deferred;
+use seregazhuk\React\PromiseTesting\TestCase;
+
+final class AboutPromiseTest extends TestCase
+{
+	/** @test */
+	public function predicate_function_reveals_what_is_true_about_promise(): void
```

Comment:
Looks like you used tabs instead of spaces here. Can you fix it, please?

Reply:
Yes, I will. Thanks for responding.

Reply:
Does this look better? https://github.com/seregazhuk/php-react-promise-testing/pull/21/commits/a5a2f7d532237f2e2f2068cb6217ca8199e111dd

Reply:
Looks great! 🔥 There is a conflict with master branch. Can you fix it? and I'll merge the request.